package-lock.json

```diff
@@ -0,0 +1,6295 @@
+{
```

Comment:
As we use `yarn` remove `package-lock.json`

Reply:
Removed.